### PR TITLE
SDK-81 gathering stats

### PR DESCRIPTION
**Pull request by AdamGrzybkowski**

### CODE REVIEW

File: integration-tests/src/test/resources/integration-test/sdkStats.properties [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#Tue Jul 12 07:41:36 CEST 2016
```
Comment (rkorytkowski) [inline integration-tests/src/test/resources/integration-test/sdkStats.properties]: Let's name the file sdk-stats.properties

File: integration-tests/src/test/resources/integration-test/sdkStats.properties [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+statsUndeployCalls=0
+statsUnwatchCalls=0
+statsLastReported=12/7/2016
+statsResetCalls=0
+statsLastUsed=12/7/2016
+statsInfoCalls=0
+statsEnabled=false
+statsDeleteCalls=0
+statsSdkVersion=3.0.0-SNAPSHOT
+statsUser=random-user-39f19146-353f-4854-9333-2c4aab6293e0
```
Comment (rkorytkowski) [inline integration-tests/src/test/resources/integration-test/sdkStats.properties]: Let's remove 'random-user' prefix

**Comment by AdamGrzybkowski** [comment]
@rkorytkowski I have changed it and now the stats are being sent to google form https://docs.google.com/forms/d/1Pm11QyavEyQhxhYp1YP1OuS1KK2b0hc3T-QE8LhYENI

File: maven-plugin/pom.xml [hunk 3]
```diff
@@ -117,6 +117,11 @@
             <groupId>javax.servlet</groupId>
             <artifactId>jstl</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.dropbox.core</groupId>
+            <artifactId>dropbox-core-sdk</artifactId>
+            <version>2.0.6</version>
+        </dependency>
```
Comment (rkorytkowski) [inline maven-plugin/pom.xml]: Do we still need dropbox dependency?

Comment (AdamGrzybkowski) [inline maven-plugin/pom.xml]: No, I forgot to delete it :)

File: maven-plugin/src/main/java/org/openmrs/maven/plugins/model/SdkStatistics.java [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+     * @throws MojoExecutionException
+     */
+    public void sendReport() throws MojoExecutionException {
+        if( checkIfOneWeekFromLastReport() && getStatsEnabled()){
+            setLastReported();
+            save();
+            try {
+                postToGoogleForm();
+            } catch (DbxException | IOException e) {
+                throw new IllegalStateException("Could not send statistics", e);
```
Comment (rkorytkowski) [inline maven-plugin/src/main/java/org/openmrs/maven/plugins/model/SdkStatistics.java]: I'd rather we fail silently if stats cannot be sent... Let's just display a short message to the user "Anonymous usage statistics could not be sent due to " +  e.getMessage() and continue execution.

Comment (AdamGrzybkowski) [inline maven-plugin/src/main/java/org/openmrs/maven/plugins/model/SdkStatistics.java]: I've updated PR

**Pull request merged**